### PR TITLE
Updates for Monojet and stuff for V-tags 

### PR DESCRIPTION
**Pull request by emanueledimarco**
- updates for monojet analysis and control regions 
- include stuff from @simoneg90 for hadronic V-tags studies

### CODE REVIEW

**Comment by mariadalfonso** [comment]
@emanueledimarco 
Since the the monojet analysis is pretty a noise catcher, some suggestion

This halo filter is pretty obsolete (you need to move to the 2016)
https://github.com/CERN-PH-CMG/cmgtools-lite/pull/98/files#diff-99d13cc27ffc16b9fa3cdd942eb80f6fR59

you can check in the https://github.com/CERN-PH-CMG/cmgtools-lite/blob/80X/ObjectStudies/python/analyzers/metCoreModules_cff.py#L78

Also you might want to add these two pretty useful flags
https://github.com/CERN-PH-CMG/cmgtools-lite/blob/80X/ObjectStudies/python/analyzers/metCoreModules_cff.py#L53
https://github.com/CERN-PH-CMG/cmgtools-lite/blob/80X/ObjectStudies/python/analyzers/metCoreModules_cff.py#L60

File: MonoXAnalysis/python/samples/samples_13TeV_VJETS_RunIISpring16MiniAODv2.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import PhysicsTools.HeppyCore.framework.config as cfg
+import os
+
+from CMGTools.RootTools.samples.ComponentCreator import ComponentCreator
+kreator = ComponentCreator()
+
+
```
Comment (mariadalfonso) [inline MonoXAnalysis/python/samples/samples_13TeV_VJETS_RunIISpring16MiniAODv2.py:12]: these BKG samples should be added in 
/RootTools/python/samples/samples_13TeV_RunIISpring16MiniAODv2.py

**Comment by emanueledimarco** [comment]
@mariadalfonso done both changes (thanks for the new met flags). 
Also, moved the new background samples added by @simoneg90 in RootTools samples with some more general naming.

File: MonoXAnalysis/python/analyzers/dmCore_modules_cff.py [hunk 2]
```diff
@@ -57,6 +73,7 @@
         "HBHENoiseFilter" : [ "Flag_HBHENoiseFilter" ],
         "HBHENoiseIsoFilter" : [ "Flag_HBHENoiseIsoFilter" ],
         "CSCTightHaloFilter" : [ "Flag_CSCTightHaloFilter" ],
+        "CSCTightHalo2015Filter" : [ "Flag_CSCTightHalo2015Filter" ],
```
Comment (mariadalfonso) [inline MonoXAnalysis/python/analyzers/dmCore_modules_cff.py:76]: In 2016 we need to apply the Flag_globalTightHalo2016Filter

**Comment by emanueledimarco** [comment]
Added. Thanks!

**Pull request merged**